config/locales/custom/activerecord.yml

```
@@ -26,4 +30,8 @@ en:
 
     attributes:
       budget/investment:
-        heading_id: "Category"
\ No newline at end of file
+        heading_id: "Category"
+      site_customization/page:
+        background_color: Background color
+        custom_header_enabled: "Enable custom header?"
```

```suggestion
        custom_header_enabled: "Enable custom header"
```

I think we usually don't add question marks to texts for boolean fields :thinking:.